ui/pages/send/send-content/send-content.component.js

```diff
@@ -86,24 +94,53 @@ export default class SendContent extends Component {
     );
   }
 
+  checkContractAddress = async () => {
+    const { recipient, tokenAddressList } = this.props;
+
+    const symbolAndDecimals = await getSymbolAndDecimals(
```

Comment:
@filipsekulic Rereading this now, I realize a made a mistake in previous advice I gave you. We shouldn't call `getSymbolAndDecimals` because that will return symbol and decimals for all contracts that have symbol and decimals APIs implemented, but we only want to show the warning when it is from our token list.

I think if you use the `getTokenMetadata` from the same file, instead of `getSymbolAndDecimals` this will work correctly. With that change, you will also just need to add an optional chaining operator below (e.g. `symbolAndDecimals?.symbol`)

Reply:
Fixed in: e3b7ee44fa5b49dbce8251dcb856212b1b34eb85